openspeedtest-server.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "OpenSpeedTest-Server",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/openspeedtest/iocage-plugin-OpenSpeedTest-Server.git",
+    "properties": {
+        "nat": 1,
+        "nat_forwards": "tcp(8080:8080)"
+    },
+    "pkgs": [
+        "nginx"
+    ],
+    "packagesite": "http://pkg.FreeBSD.org/FreeBSD:11:amd64/latest",
```

Comment:
Could this FreeBSD:11:amd64 be changed to ${ABI} that way on 12.0 it fetches packages from 12 repo?